app/views/user/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>User</h1>
-<p>Here is the list of user for the blog app</p>
+<% if @users.blank? %>
+<p>There are not books</p>
+<% else %>
+<p>These are current Users.</p>
+
+<ul>
+<% @users.each do |u| %>
+<li>
+ <img src="<%= u.photo %>" alt="image">
+ <h2><%= link_to u.name, {:action => 'show', :id => u.id}  %></h2>
+ <p>Number of posts: <%= u.posts.length %> </p>
+  </li>
+<% end %>
+</ul>
+<button>See all the post</button>
```

Comment:
- [OPTIONAL] It would be nice to make this a partial and render it on this page, the user's show page and the post's index page. To implement that, you can transfer these codes to a _**partial**_ and render it on these pages. 

`<%= render @users %>` 

You can learn more about _**partials**_ in this [article](https://edgeapi.rubyonrails.org/classes/ActionView/PartialRenderer.html)